test/LEGO.AsyncAPI.Tests/Bindings/Sqs/SqsBindings_should.cs

```diff
@@ -23,6 +23,8 @@ public void SqsChannelBinding_WithFilledObject_SerializesAndDeserializes()
     queue:
       name: myQueue
       fifoQueue: true
+      deduplicationScope: messageGroup
```

Comment:
as they are nullable please make a test where they are missing

Reply:
This is represented by the dead letter queue currently - https://github.com/LEGO/AsyncAPI.NET/pull/135/files#diff-7e55e98b98b0a01be4283763392ee8d80174d8b84cfb01068392b785327794e1L178. But I'm happy to add an explicit test for the main queue if you think it would be useful?

Reply:
If its already represented, then i am all good.

Reply:
I've amended one of the tests to check for when no FIFO input is provided, as previously both input tests use FIFO input for the main queue. Just for better coverage